Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json

```diff
@@ -517,7 +516,7 @@
                         "y": 14
                     },
                     {
-                        "description": "",
+                        "description": ":warning: Links may be malicious!",
```

Comment:
Quesiton: is that presented by defualt? 
I mean we have an option to hide out the description so I hope we turned that on!

Reply:
```suggestion
                        "description": ":warning: This is the original HTML Email Section. Clicking on Links and URLs can be lead to a malicious site! ",
```

Reply:
The hiding option is for the section name, but the description will show